src/models/Course/Course.tsx

```diff
@@ -0,0 +1,12 @@
+export default class Course {
+    id: number;
+    name: string;
+    link: string;
+    
+  
+    constructor() {
+      this.id = 0;
+      this.name = "";
+      this.link = "";
+    }
+  }
```

Comment:
In Typescript better use Interface if you don'n have any methods. And we try to always use interface in `/Models`